recipes/combo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "combo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3c5da2b8af17e03146ce382d10150373a260ce64565963f7c0b0cecfc7d093be
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numba >=0.35
+    - numpy >=1.13
+    - python
+    - scikit-learn >=0.20
+    - scipy
```

Comment:
I might be wrong, but it seems this project also depends on ``pyod``, am I right?

Reply:
I think there was a circular dependency and pyod was optional

Reply:
It's optional: https://github.com/conda-forge/staged-recipes/pull/10964#issuecomment-594653410